pkg/controller/status/status.go

```diff
@@ -475,8 +494,8 @@ func (m *statusManager) containerErrorMessage(p corev1.Pod, c corev1.ContainerSt
 }
 
 func (m *statusManager) set(conditions ...operator.TigeraStatusCondition) {
-	if !m.enabled {
-		// Never set any conditions unless the status manager is enabled.
+	if m.enabled == nil || !*m.enabled {
+		// Never set any conditions unless the CR is present.
```

Comment:
I think this should still say "is enabled". 

At the moment, we know we set this bit based on CR, but from the context of the status manager this variable doesn't signify if the CR is present, rather it signifies:

- enabled
- disabled
- waiting-for-information.